### PR TITLE
test: test_create_keycard_account_with_new_seed_phrase added

### PR DESCRIPTION
**Pull request by Valentina1133**
As we decided to replace image comparison with other verifications I will add verifications for keycard images in separate task.

Screenshot 2023-10-31 at 13 52 10

### CODE REVIEW

**Review by anastasiyaig — COMMENTED** [review]
i feel it is a good idea to keep 1 test file per test case. If u agree (yes it will result in multiple files but easier to work with), then can u please rename this certain file as the test case (or similar)? and other tests will live in separate files
Our tests a really long (a lot of lines) and having more than 2 in one file makes it hard to read. I will do the same for existing wallet tests so we will have a better structure

File: gui/components/settings/keycard_popup.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    @allure.step('Create keycard account with seed phrase')
+    def create_keycard_account_with_seed_phrase(self, keycard_name: str, account_name: str):
+        time.sleep(1)
+        self.click_next().reveal_seed_phrase()
+        seed_phrases = self.get_seed_phrases
+        self.click_next()
+        self.confirm_first_word(seed_phrases).confirm_second_word(seed_phrases).confirm_third_word(seed_phrases)
+        self.click_next().name_keycard(keycard_name)
+        self.click_next().name_account(account_name)
+        self.click_next()
```
Comment (anastasiyaig) [inline gui/components/settings/keycard_popup.py:130]: +1 extra line needed perhaps?

**Review by anastasiyaig — APPROVED** [review]

File: gui/components/settings/keycard_popup.py (hunk 1 above)
Comment (Valentina1133) [reply gui/components/settings/keycard_popup.py:130]: There is one)

File: tests/settings/settings_keycard/test_keycard.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+import configs
+import driver
+from constants import ColorCodes
+from constants.keycard import Keycard
+from gui.main_window import MainWindow
+from gui.mocked_keycard_controller import MockedKeycardController
+from scripts.tools import image
+
+pytestmark = allure.suite("Keycard")
```
[the remark this answers is not in the document]
Comment (Valentina1133) [reply tests/settings/settings_keycard/test_keycard.py:13]: Done

**Comment by Valentina1133** [comment]
> i feel it is a good idea to keep 1 test file per test case. If u agree (yes it will result in multiple files but easier to work with), then can u please rename this certain file (tests/settings/settings_keycard/test_keycard.py) as the test case (or similar)? and other tests will live in separate files
> Our tests a really long (a lot of lines) and having more than 2 in one file makes it hard to read. I will do the same for existing wallet tests so we will have a better structure

Ok. I renamed it

File: gui/objects_map/component_names.py [hunk 3]
```diff
@@ -317,3 +317,21 @@
...
+nextStatusButton = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "objectName": "PrimaryButton", "type": "StatusButton", "visible": True}
+revealSeedPhraseButton = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "objectName": "AddAccountPopup-RevealSeedPhrase", "type": "StatusButton", "visible": True}
+seedPhraseWordAtIndex_Placeholder = {"container": statusDesktop_mainWindow_overlay, "objectName": "SeedPhraseWordAtIndex-%WORD-INDEX%", "type": "StatusSeedPhraseInput", "visible": True}
+word0_StatusInput = {"container": statusDesktop_mainWindow_overlay, "id": "word0", "type": "StatusInput", "unnamed": 1, "visible": True}
+word1_StatusInput = {"container": statusDesktop_mainWindow_overlay, "id": "word1", "type": "StatusInput", "unnamed": 1, "visible": True}
+word2_StatusInput = {"container": statusDesktop_mainWindow_overlay, "id": "word2", "type": "StatusInput", "unnamed": 1, "visible": True}
+edit_TextEdit = {"container": statusDesktop_mainWindow_overlay, "type": "TextEdit", "unnamed": 1, "visible": True}
```
[the remark this answers is not in the document]
Comment (Valentina1133) [reply gui/objects_map/component_names.py]: Fixed

**Review by StateOf-Vlado — APPROVED** [review]

**Pull request merged**